python/kdumpfile.c

```diff
@@ -101,35 +103,59 @@ kdumpfile_dealloc(PyObject *_self)
 	self->ob_type->tp_free((PyObject*)self);
 }
 
+static PyObject *exception_map(kdump_status status)
+{
+	switch (status) {
+	case kdump_ok:		return NULL;
+	case kdump_syserr:	return PyExc_SystemError;
+	case kdump_unsupported: return PyExc_TypeError;
+	case kdump_nodata:	return PyExc_IndexError;
+	case kdump_dataerr:	return PyExc_IOError;
+	case kdump_invalid:	return PyExc_IOError;
```

Comment:
This status code is meant for invalid arguments to a function (e.g. specifying an out-of-range value as address space, or trying to enumerate a leaf argument). It seem to me that `ValueError` would be more appropriate.